games/static/games/proto.js

```diff
@@ -9,50 +9,52 @@ class Game {
   }
   start() {
     let gameId = 0;
-    let cleanId = 0;
-    let start
-    let restart
-    let board = new Board(this.grid, BOARD_WIDTH, this.setup);
-    this.startButton.addEventListener("mousedown", start = () => {
+    let startButtonEvent = () => {
       if (gameId) {
         clearInterval(gameId);
         gameId = null;
         this.gamePause();
-      } else {
-          this.gameRun();
-          gameId = setInterval(() => {
-            if (board.noAliensLeft()){
-              this.grid.innerHTML = '';
-              this.grid.classList.add('youwin');
-              clearInterval(gameId);
-              this.gamePause();
-            } 
-            board.moveAliens();
-          }, 500);
-      }
-    });
+        return 0
+      } 
+      this.gameRun();
+      gameId = setInterval(() => {
+        if (board.isAlienFree   ()){
+          this.announceWinner()
+          clearInterval(gameId);
+          this.gamePause();
+        } 
+        board.moveAliens();
+      }, 500);
+    }
+    let restartButtonEvent = () => {
```

Comment:
rename to `restartButtonEventHandler`
the word handler usually used with those kind of things of button events.
you will see it a lot
lets act like romans in rome ;-)